frontend/dashboard/src/RelayEnvironment.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from "relay-runtime";
 import keycloak from "./keycloak";
 
-const HTTP_ENDPOINT = "https://workflows.diamond.ac.uk/graphql";
+const HTTP_ENDPOINT = "http://workflows.diamond.ac.uk/graphql";
```

Comment:
should probably be https?

Reply:
I've put it back to https. There's a CORS-related warning that this won't work with https soon, so something to look into.